var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -63,19 +87,19 @@ def cmake_args(self):
             '-DGEANT4_INSTALL_DATA=ON',
             '-DGEANT4_BUILD_TLS_MODEL=global-dynamic',
             '-DGEANT4_BUILD_MULTITHREADED=ON',
-            '-DGEANT4_USE_USOLIDS=ON',
             '-DGEANT4_USE_SYSTEM_EXPAT=ON',
             '-DGEANT4_USE_SYSTEM_ZLIB=ON',
             '-DXERCESC_ROOT_DIR:STRING=%s' %
-            spec['xerces-c'].prefix,
-            '-DUSolids_DIR=%s' %
-            join_path(spec['vecgeom'].prefix, 'lib/CMake/USolids')]
+            spec['xerces-c'].prefix, ]
```

Comment:
You don't need `prefix.lib.CMake.USolids`?

Reply:
I do, but it got moved to https://github.com/OSGConnect/spack/blob/develop/var/spack/repos/builtin/packages/geant4/package.py#L117-L118